app/components/card-hazard.tsx

```diff
@@ -6,14 +6,23 @@ import {
   Text,
   HStack,
 } from "@chakra-ui/react";
-import { Hazards } from "./__mocks__/hazards";
 import Pill from "./pill";
 
-const CardHazard = () => {
-  const hazard = Hazards[0];
+interface CardHazardProps {
+  hazard: {
+    id: number;
+    name: string;
+    title: string;
+    description: string;
+    update: string;
+    color: string;
+  };
+}
 
+const CardHazard: React.FC<CardHazardProps> = ({ hazard }) => {
+  const { title, description, update, color } = hazard;
```

Comment:
Nit: If you won't use `hazard` directly (see other comment below), you could consider destructuring in the parameters

Reply:
Good point—updated to destructure the parameter as suggested.